### PR TITLE
tverigrad.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/tverigrad.ru.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<ruleset name="tverigrad.ru">
+	<target host="tverigrad.ru" />
+	<target host="www.tverigrad.ru" />
+	<!-- <target host="m.tverigrad.ru" /> --><!-- ERROR (35, schannel: SNI or certificate check failed: SEC_E_WRONG_PRINCIPAL (0x80090322) -    .) -->
+	<!-- <target host="client.tverigrad.ru" /> --><!-- ERROR (35, schannel: SNI or certificate check failed: SEC_E_WRONG_PRINCIPAL (0x80090322) -    .) -->
```
Comment (J0WI) [inline src/chrome/content/rules/tverigrad.ru.xml]: eklama.tverigrad.ru

**Comment by galeksandrp** [comment]
Please merge #6760, then I rebase this.

**Comment by J0WI** [comment]
Why did you removed comments for `m.tverigrad.ru` and `client.tverigrad.ru`?

**Comment by galeksandrp** [comment]
Added.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**